lib/ruboclean/runner.rb

```diff
@@ -13,16 +13,28 @@ def initialize(args = [])
     def run!
       return if source_file_pathname.empty?
 
-      load_file.then(&method(:order))
-               .then(&method(:cleanup_paths))
-               .then(&method(:convert_to_yaml))
-               .then(&method(:write_file!))
+      ordered_file = load_file
+                     .then(&method(:order))
+                     .then(&method(:cleanup_paths))
+                     .then(&method(:convert_to_yaml))
```

Comment:
Yes! I also like chaining `.then`s. I considered using `#tap` instead of `#then` in chain to simplify `write_file!`. End up I decided to use `#then` as you say. It looks more pretty!